modules/fileout_netcdf/FONcArray.cc

```diff
@@ -581,7 +592,16 @@ void FONcArray::define(int ncid) {
             int dimnum = 0;
             for (; i != e; i++) {
```

Comment:
This loop could be a range-based for-loop.

Reply:
Somehow the iterartor doesn't like the range-based for-loop. So just keep the current code.

Reply:
I have another ticket to clean up fileout netCDF. I will remove #if 0 #endif and other things when I work on that ticket